docs/proposals/kubeedgeadm-scope.md

```diff
@@ -199,7 +211,7 @@ Flags:
     4. It will update the certificate information in `controller.yaml`
     5. Start `kubeadm init`.
 
-       **NOTE:** If any issues or error reported from `kubeadm init`, `kubeedge cloud init` command shall be not responsible, as it may occur due to the environment. User has to resolve it. We are taking up this activity to have 1 click install approach for KubeEdge and not K8S.
+       **NOTE:** If any issues or error reported from `kubeadm init` command shall be not responsible, as it may occur due to the environment. User has to resolve it. We are taking up this activity to have 1 click install approach for KubeEdge and not K8S.
```

Comment:
If any issues or error reported from `kubeadm init` command shall be not responsible, as it may occur due to the environment.  --->

Issues encountered while performing `kubeadm init` need to be resolved by the user.